lib/fog/vsphere/models/compute/server.rb

```diff
@@ -50,11 +51,14 @@ class Server < Fog::Compute::Server
         attribute :memoryHotAddEnabled
         attribute :firmware
 
+        has_many_identities :cdroms, :cdroms
+
         def initialize(attributes={} )
           super defaults.merge(attributes)
           self.instance_uuid ||= id # TODO: remvoe instance_uuid as it can be replaced with simple id
           initialize_interfaces
           initialize_volumes
+          initialize_cdroms
```

Comment:
You don't need to initialize anymore because the association DSL will do that for you.